docs/gettingstarted/installation-windows.rst

```diff
@@ -45,7 +45,7 @@ Make sure that "**Set up PATH environment variable to include GTK+**" is selecte
 Compilers
 ~~~~~~~~~
 
-Please download and install the latest version of `Visual C++ build tools <https://landinghub.visualstudio.com/visual-cpp-build-tools>`_.
+Please download and install the latest version of `Visual C++ build tools <https://go.microsoft.com/fwlink/?linkid=840931>`_.
```

Comment:
I don't think the link label is accurate now. The new package seems to be named "Build Tools for Visual Studio 2017".